File: Serene/Serene.Script/Meeting/MeetingAgendaType/MeetingAgendaTypeGrid.cs

namespace Serene.Meeting
{
    using jQueryApi;
    using Serenity;
    using System;
    using System.Collections.Generic;
    using System.Runtime.CompilerServices;

    [ColumnsKey("Meeting.MeetingAgendaType"), IdProperty("MeetingAgendaTypeId"), NameProperty("Name"), IsActiveProperty("IsActive")]
    [DialogType(typeof(MeetingAgendaTypeDialog)), LocalTextPrefix("Meeting.MeetingAgendaType"), Service("Meeting/MeetingAgendaType")]
    public class MeetingAgendaTypeGrid : EntityGrid<MeetingAgendaTypeRow>
    {
        public MeetingAgendaTypeGrid(jQueryObject container)
            : base(container)
        {
        }
    }

    // Please remove this partial class or the first line below, after you run ScriptContexts.tt
    [Imported, Serializable, PreserveMemberCase] 
    public partial class MeetingAgendaTypeRow
    {
    }
}

File: Serene/Serene.AspNetCore/Modules/Meeting/MeetingAgendaType/MeetingAgendaTypeGrid.ts

namespace Serene.Meeting {
    
    @Serenity.Decorators.registerClass()
    export class MeetingAgendaTypeGrid extends Serenity.EntityGrid<MeetingAgendaTypeRow, any> {
        protected getColumnsKey() { return 'Meeting.MeetingAgendaType'; }
        protected getDialogType() { return MeetingAgendaTypeDialog; }
        protected getIdProperty() { return MeetingAgendaTypeRow.idProperty; }
        protected getLocalTextPrefix() { return MeetingAgendaTypeRow.localTextPrefix; }
        protected getService() { return MeetingAgendaTypeService.baseUrl; }

        constructor(container: JQuery) {
            super(container);
        }
    }
}